core/src/main/scala/chisel3/experimental/VerificationIntrinsics.scala

```diff
@@ -0,0 +1,33 @@
+package chisel3.experimental
+
+import chisel3._
+import scala.reflect.macros.blackbox.Context
+import scala.language.experimental.macros
+import chisel3.internal._
+import chisel3.internal.Builder.pushCommand
+import chisel3.internal.firrtl.ir._
+import chisel3.experimental.{BaseModule, SourceInfo}
+
+import scala.annotation.nowarn
+import scala.reflect.macros.blackbox
+import chisel3.experimental.IntrinsicModule
+
+/** Create a module for `ifelsefatal` style assertion.
+  */
+private[chisel3] class Chisel_IfElseFatal[T <: Data](
```

Comment:
Not sure about this underscore naming pattern

Reply:
Thanks, I'll change it to `IfElseFatalIntrinsic`.